e2e/LongHaul/service/HubEvents.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Consumer;
+
+namespace Microsoft.Azure.Devices.LongHaul.Service
+{
+    internal class HubEvents
+    {
+        private readonly IotHub _iotHub;
+
+        internal HubEvents(IotHub serviceClient)
+        {
+            _iotHub = serviceClient;
+        }
+
+        internal async Task RunAsync(CancellationToken ct)
```

Comment:
We can rework this when we're ready to have callbacks or something so we can respond to events in the IotHub class.